alpenhorn/client.py

```diff
@@ -1090,15 +1087,13 @@ def import_files(node_name, verbose, acq, dry):
 
 
 def get_e2label(dev):
-    pin, pout, perr = os.popen3("/sbin/e2label %s" % dev, "r")
-    pin.close()
-    res = pout.read().strip()
-    err = perr.read()
-    pout.close()
-    perr.close()
-    if not len(err) and len(res) < MAX_E2LABEL_LEN:
-        return res
-    return None
+    try:
+        return subprocess.check_output(
+            args=["/sbin/e2label", dev],
```

Comment:
This looks good in theory!

What is `dev` going to look like (what is the expect structure of the argument passed to `get_e2label`)?

Reply:
It's just the path to the partition (e.g. `"/dev/sda1"`)